.gitmodules

```diff
@@ -0,0 +1,12 @@
+[submodule "submodules/patternfly-core"]
+	path = submodules/patternfly-core
+	url = https://github.com/patternfly/patternfly.git
+	branch = master
```

Comment:
We should pull from master-dist, similar to angular-patternfly, so pf.org can ultimately remove the patternfly dependency.